cmd/fabric8-jenkins-proxy/main.go

```
@@ -106,12 +106,30 @@ func start(config configuration.Configuration, tenant *clients.Tenant, wit clien
 	ctx, cancel := context.WithCancel(context.Background())
 	defer cancel()
 
-	startWorkers(ctx, &wg, cancel, store, &proxy, defaultStatsLoggingInterval, config.GetDebugMode())
+	startWorkers(ctx, &wg, cancel, store, &proxy, defaultStatsLoggingInterval, config)
 	setupSignalChannel(cancel)
 	wg.Wait()
 }
 
-func startWorkers(ctx context.Context, wg *sync.WaitGroup, cancel context.CancelFunc, store storage.Store, proxy *proxy.Proxy, interval time.Duration, addProfiler bool) {
+func listenAndServe(srv *http.Server, cancel context.CancelFunc, enableHTTPS bool) {
+	if enableHTTPS {
+		if err := srv.ListenAndServeTLS("server.crt", "server.key"); err != nil {
```

need to log error otherwise this fails silently